libnl-route-3.sym

```diff
@@ -843,10 +843,14 @@ global:
 	rtnl_class_hfsc_set_usc;
 	rtnl_link_inet6_addrgenmode2str;
 	rtnl_link_inet6_get_addr_gen_mode;
+	rtnl_link_inet6_get_flags;
```

Comment:
I moved the functions to the new section libnl_3_4 as successor for section libnl_3_2_28. Is this correct?